docker.d/configs/scriptworker.yaml

```diff
@@ -0,0 +1,56 @@
+provisioner_id: 'scriptworker-prov-v1'
+worker_group: 'treescriptworker-v1'
+worker_type:
+  $if: 'COT_PRODUCT == "thunderbird"'
+  then:
+    $if: 'ENV == "prod"'
+    else: 'tb-treescript-comm-v1'
+    then: 'tb-treescript-comm-dev'
+  else:
+    $if: 'ENV == "prod"'
+    else: 'treescript-v1'
```

Comment:
it should be if-then-else (it's is if-else-then now)

Reply:
oh my god .. i was doing this in a hurry. sorry for review time wasted i should have take a better look myself.